### ISSUE TITLE
feat: Onboarding screen

### ISSUE THREAD

**Issue by anogire** [BinaryStudioAcademy/bsa-2024-bebalance#127]
### What feature?

- [x] Create Onboarding component with Header and Content.
- [x] For Header use `react-native-progress-step-bar` library.
- [x] The content will need to use our custom RadioGroup component with answers to questions.
- [x] Add 'Next' button as on web design to go to the next question
- [x] For changing questions it's possible to use `react-native-infinite-pager`
- [x] When you select the answer to the last question - you go to the “Welcome” screen.

### Add screenshots

<img width="354" alt="Screenshot 2024-08-22 at 17 21 46" src="https://github.com/user-attachments/assets/baee8b83-2f85-4430-8b31-aa65dafdf33a">

**Comment by KotenkoNata** [comment]
For Header use react-native-progress-step-bar library - Do I need create Header or is it better create progress bar instead of Header?
Would you like me to use some mock data for Content part - title and question?

**Comment by MarinaBershova** [comment]
1. The text of questions should be in bold
2. The text of the answers should be centered
3. Need to change the paddings between question and answers, and between answers (according to the design)


![Image](https://github.com/user-attachments/assets/47748df9-fb0d-418a-8d66-fcb9c5dff1d2)


4. Need to add padding between text and border


![Image](https://github.com/user-attachments/assets/0e74bbec-96b8-41b7-814c-a183298c3800)

**Comment by KotenkoNata** [comment]
I had a [task](https://github.com/orgs/BinaryStudioAcademy/projects/29/views/1?sliceBy%5Bvalue%5D=KotenkoNata&pane=issue&itemId=76008112) that described align of text inside buttons.

**Comment by revenets** [comment]
@MarinaBershova we are ok with the text alignment inside the answer radio button. Paddings and tex wrapping inside answer option will be fixed

**Comment by KotenkoNata** [comment]
@revenets  I've fixed the margins and paddings to the size of the question title. 

I propose fixing buttons at the bottom, as in the example below.

[fixed.webm](https://github.com/user-attachments/assets/87e696ac-1c1f-4059-9601-2df39a414602)

How perfect should it be a realization to design? 
I've made the gap between buttons less than in Figma because, in the case of 25, there is no space for buttons. 
Is it ok?
